package.json

```diff
@@ -68,6 +68,7 @@
     "next": "12.2.0",
     "papaparse": "^5.3.2",
     "qrcode": "^1.5.1",
+    "qrcode.react": "^3.1.0",
```

Comment:
Is there a reason we are not reusing the existing `qrcode` library to render the qr code as an image?

Reply:
`qrcode` decodes QR codes, whereas `qrcode.react` generates them.

Reply:
Migrating code 🙈 Looks like we should entirely switch from `qrcode` to `qrcode.react` though as it is still maintained/allows for more features. I would tackle this in a separate PR though.